.github/workflows/module-checks.yml

```diff
@@ -3,11 +3,11 @@ name: Module Checks
 on:
   push:
     branches: ["main"]
-    paths: ['modules/**', '.github/workflows/module-checks.yml']
+    paths: ["modules/**", "requirements-dev.txt", ".github/workflows/module-checks.yml"]
```

Comment:
Updates to `requirements-dev.txt` can change how our linters behave, which could lead to errors in our validation scripts.